lib/ansible/modules/cloud/amazon/inspector_target.py

```diff
@@ -0,0 +1,238 @@
+      - Required if C(state=present).
+extends_documentation_fragment:
+  - aws
+  - ec2
+requirements:
+  - boto3
+  - botocore
+'''
+
+EXAMPLES = '''
+- name: Create my_target Assessment Target
+  inspector_target:
+    name: my_target
+    tags:
+      role: scan_target
+
+- name: Update Existing my_target Assessment Target with Additional Tags
+  inspector_target:
+    name: my_target
+    tags:
+      env: dev
+      role: scan_target
+
+- name: Delete my_target Assessment Target
+  inspector_target:
+    name: my_target
+    state: absent
+'''
+
+RETURN = '''
+arn:
+  description: The ARN that specifies the Amazon Inspector assessment target.
+  returned: success
+  type: string
+  sample: "arn:aws:inspector:eu-west-1:123456789012:target/0-O4LnL7n1"
+created_at:
+  description: The time at which the assessment target was created.
+  returned: success
+  type: string
+  sample: "2018-01-29T13:48:51.958000+00:00"
+name:
+  description: The name of the Amazon Inspector assessment target.
+  returned: success
+  type: string
+  sample: "my_target"
+resource_group_arn:
+  description: The ARN that specifies the resource group that is associated
+               with the assessment target.
+  returned: success
+  type: string
+  sample: "arn:aws:inspector:eu-west-1:123456789012:resourcegroup/0-qY4gDel8"
+tags:
+  description: The tags of the resource group that is associated with the
+               assessment target.
+  returned: success
+  type: list
+  sample: [{"key": "role", "value": "scan_target"},
+           {"key": "env", "value": "dev"}]
+updated_at:
+  description: The time at which the assessment target was last updated.
+  returned: success
+  type: string
+  sample: "2018-01-29T13:48:51.958000+00:00"
+'''
+
+try:
+    from botocore.exceptions import ClientError, ValidationError
+except ImportError:
+    pass  # Handled AnsibleAWSModule
+from ansible.module_utils.aws.core import AnsibleAWSModule
+from ansible.module_utils.ec2 import AWSRetry
+from ansible.module_utils.ec2 import (
+    boto3_conn,
+    camel_dict_to_snake_dict,
+    get_aws_connection_info,
+)
+
+
+@AWSRetry.backoff()
+def main():
+    argument_spec = dict(
+        name=dict(required=True),
+        state=dict(choices=['absent', 'present'], default='present'),
+        tags=dict(type='dict'),
+    )
+
+    required_if = [['state', 'present', ['tags']]]
+
+    module = AnsibleAWSModule(
+        argument_spec=argument_spec,
+        supports_check_mode=False,
+        required_if=required_if,
+    )
+
+    name = module.params.get('name')
+    state = module.params.get('state').lower()
+    tags = module.params.get('tags')
+    if tags:
+        tags = [{'key': key, 'value': value} for key, value in tags.items()]
+
+    region, ec2_url, aws_connect_kwargs = get_aws_connection_info(
+        module,
+        boto3=True,
+    )
+
+    if not region:
+        module.fail_json(msg='region must be specified')
+
+    try:
+        client = boto3_conn(
```

Comment:
This doesn't need to be in a try/except. Since you're using AnsibleAWSModule, you can replace boto3_conn with: `client = module.client('inspector')` and remove the call to get_aws_connection_info above.